roles/ceph-facts/tasks/facts.yml

```diff
@@ -249,6 +249,22 @@
   set_fact:
     osd_pool_default_min_size: "{{ ceph_conf_overrides.get('global', {}).get('osd_pool_default_min_size', ceph_osd_pool_default_min_size) }}"
 
+- name: check if the ceph conf exists
+  stat:
+    path: '/etc/ceph/{{ cluster }}.conf'
+  register: ceph_conf
+
+- command: grep 'osd pool default crush rule' /etc/ceph/{{ cluster }}.conf
```

Comment:
add `- name: get 'osd pool default crush rule' value from /etc/ceph/{{ cluster }}.conf` or something similar please

Reply:
Sure I'll add that